openhabian-setup.sh

```diff
@@ -498,6 +498,29 @@ frontail() {
   if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED (service)"; exit 1; fi
 }
 
+nodered() {
+  echo -n "$(timestamp) [openHABian] Installing Node-RED... "
+  if ! command -v npm &>/dev/null; then
+    cond_redirect wget -O - https://deb.nodesource.com/setup_7.x | bash -
+    if [ $? -ne 0 ]; then echo "FAILED (prerequisites)"; exit 1; fi
+    cond_redirect apt -y install nodejs
+    if [ $? -ne 0 ]; then echo "FAILED (nodejs)"; exit 1; fi
+  fi
```

Comment:
Second place where we need nodejs as an dependency. Let's move it to it's own function. wdyt?